pkgs/applications/networking/irc/weechat/scripts/weechat-matrix/default.nix

```
@@ -0,0 +1,59 @@
+{ buildPythonPackage, stdenv, python, fetchFromGitHub,
+  pyopenssl, typing, webcolors, future, atomicwrites,
+  attrs, Logbook, pygments, cachetools, matrix-nio }:
+
+let
+  matrixUploadPython = python.withPackages (ps: with ps; [
+    magic
+  ]);
+in buildPythonPackage {
+  pname = "weechat-matrix";
+  version = "unstable-2019-11-10";
+
+  src = fetchFromGitHub {
+    owner = "poljar";
+    repo = "weechat-matrix";
+    rev = "69ad2a9c03d516c212d3d0700dbb2bfe654f6365";
+    sha256 = "1mfbkag5np2lgv6f31nyfnvavyh67jrrx6gxhzb8m99dd43lgs8c";
+  };
+
+  propagatedBuildInputs = [
+    pyopenssl
+    typing
```

this can actually cause issues as the typing package and the python3.5 stdlib typing package can conflict. To reflect the requirements.txt, i would do:
```suggestion
    ++ (if isPy27 then [ typing ] else [ aiohttp]);
```

---

I chose to remove typing entirely, this expression is only called with python3, and our weechat is only compiled with python3

---

I'm also okay with this